bash/compile_protos.sh

```diff
@@ -5,5 +5,6 @@ PROTO_DIR="./proto"
 for filename in $PROTO_DIR/*.proto; do
   lib/protobuf/src/protoc --proto_path=$PROTO_DIR \
   --cpp_out=$PROTO_DIR/cpp/ $filename
+  lib/protobuf/src/protoc --proto_path=$PROTO_DIR \
+  --python_out=$PROTO_DIR/py/ $filename
```

Comment:
make sure to run 2to3 (should be installed through pip) otherwise python will complain:

after the for loop:

2to3 --output-dir=$PROTO_DIR/py/ -W -n $PROTO_DIR/py/